src/main/java/net/pms/encoders/FFmpegAudio.java

```diff
@@ -142,7 +144,7 @@ public synchronized ProcessWrapper launchTranscode(
 		configuration = (DeviceConfiguration)params.mediaRenderer;
 		final String filename = dlna.getFileName();
 		params.maxBufferSize = configuration.getMaxAudioBuffer();
-		params.waitbeforestart = 2000;
+		params.waitbeforestart = 1;
```

Comment:
No waiting delay while playing music.
Tested by us without any issues since some months, also with my very old CPU.
If i remember correctly, we are talking about 100MB to be buffereized in 2s, it seem out of age, also for me.